orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -300,6 +294,7 @@
              </param>
              {%- endif %}
              {% if 'HUE' in exposed and 'HUE_LOAD_BALANCER' in salt['pillar.get']('gateway:location') -%}
+             <param>
```

Comment:
Is this extra param needed?

Reply:
ugh, good catch, that looks an artifact of rebase. I will fix it and resubmit the patch. Thanks